net/ntopng/src/opnsense/mvc/app/controllers/OPNsense/Ntopng/forms/general.xml

```diff
@@ -8,7 +8,9 @@
     <field>
         <id>general.interface</id>
         <label>Interfaces</label>
-        <type>dropdown</type>
+        <style>tokenize</style>
```

Comment:
@mimugmail are you sure you want a tokenizer with allow new here? this sounds like a rather static list of options.

Reply:
It works and looks nice :) Ok, I'll revert it